Tests/AmplitudeTests.m

```diff
@@ -15,12 +15,21 @@
 #import "AMPDeviceInfo.h"
 #import "AMPUtils.h"
 #import "AMPTrackingOptions.h"
+#import "AMPStorage.h"
 
 @interface Amplitude (Tests)
 
+@property (nonatomic, assign) BOOL updatingCurrently;
```

Comment:
this can be removed after we decide to not use this property in test

Reply:
Removed.